### ISSUE TITLE
Deflate all monetary values to same year

### ISSUE THREAD

**Issue by joseph-s-shapiro** [joseph-s-shapiro/erc#6]
The paper/tables text says most values are deflated to $2017 but some say they're deflated to $2018. I don't think it really matters as long as they're the same year. If we were starting the revision today I would deflate everything to the most recent year that has the relevant deflator, which I'm guessing would be 2021 or maybe 2022. But I'm fine sticking with 2017 or 2018, as long as everything's deflated to the same year. 

Could you please do the following?
- check which tables/figures/text entries claim they're deflated to $2018
- check the programs that generate these tables/figures to see if they are indeed deflated to $2018
- If so, re-run and output these results so that everything in the paper is deflated to the same year ($2017)
- Update any paper text referring to these updated tables to make sure it is consistent

@reed-walker

**Comment by rayhkim** [comment]
I checked through the code and paper, and it looks like the paper was mistaken when it said 2018 dollars—everything in the code produces numbers at 2017 dollars. I also edited the comments in the code and labels in corresponding figures/tables to reflect this. One minor detail is that AP3 uses a GDP deflator for its VSL which causes 7.4m in 2006$ to become 8.856m in 2017$, while the GDP deflator we use for marginal abatement costs uses a deflator which turns this to 8.878m in 2017$, which is a minor difference.
